docs/typescript.mdx

```diff
@@ -39,11 +39,20 @@ To make the css prop work with pure TypeScript (without babel plugin) you need t
 
 ```tsx
 /** @jsx jsx */
-import { css, jsx } from '@emotion/core'
-```
+import { jsx } from '@emotion/core'
 
+<div css={{ background: 'black' }} />
+```
 As a result you may be not able to use react fragment shorthand syntax - `<></>`, but still you can use `<Fragment></Fragment>`.
```

Comment:
we could mention here that this is actually a TS compiler limitation - in Babel it's possible to configure jsx pragma and jsxFrag pragma separately

Reply:
Done